libvast/vast/concept/printable/std/chrono.hpp

```diff
@@ -48,22 +46,47 @@ struct duration_printer : printer<duration_printer<Rep, Period, Policy>> {
 
   template <class R, class P>
   static std::string units(std::chrono::duration<R, P>) {
-    return date::detail::get_units<char>(Period{}).data();
+    if constexpr (Period::type::den == 1)
+      return "not implemented";
+    else
+      return "not implemented";
   }
 
-  template <class R>
-  static std::string units(std::chrono::duration<R, std::micro>) {
-    return "us";
+#define UNIT_SUFFIX(type, suffix)                                              \
+  template <class R>                                                           \
+  static std::string units(std::chrono::duration<R, type>) {                   \
+    return suffix;                                                             \
   }
 
+  UNIT_SUFFIX(std::atto, "as")
```

Comment:
Seems like the indentation should be a bit less.

Reply:
this is inside `struct duration_printer`, and clang-format wants it that way.

Reply:
Fair 'nuf.